content/sql/concepts/commands/terms/top/top.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: 'TOP'
+Description: 'Returns a specified number of rows from the top of the result.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Database'
+  - 'Queries'
+  - 'SQL Server'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+Returns a specified number of rows from the top of the result.
+
+## Syntax
+
+```sql
```

Comment:
The syntax section needs a few changes:
- The examples should be in a ```pseudo block.
- The examples should be generalized, such as `SELECT TOP (n)` `SELECT TOP (n) PERCENT`
- Each example should have an explanatory sentence describing what it's doing and what `n` is.

Reply:
I would also include a sentence in the "Syntax" section that explains how `column_name(s)` is a space-separated list of existing columns that are selected from `table_name`.